server/plugin/command.go

```diff
@@ -552,6 +553,16 @@ func (p *Plugin) handleWebhookAdd(_ *plugin.Context, parameters []string, args *
 	ctx := context.Background()
 	githubHook, _, err := p.CreateHook(ctx, githubClient, owner, repo, hook)
 	if err != nil {
+		if repo == "" {
+			scopes, err := p.getOauthTokenScopes(userInfo.Token.AccessToken)
```

Comment:
1/5 that the `userInfo` should be passed around here instead of the token directly.